Mage.Sets/src/mage/cards/b/BessSoulNourisher.java

```diff
@@ -0,0 +1,107 @@
+package mage.cards.b;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.counters.CounterType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.filter.predicate.mageobject.PowerPredicate;
+import mage.filter.predicate.mageobject.ToughnessPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Alex-Vasile
+ */
+public class BessSoulNourisher extends CardImpl {
+
+    static final FilterCreaturePermanent filter = new FilterCreaturePermanent("one or more other creatures with base power and toughness 1/1");
+
+    static {
+        filter.add(AnotherPredicate.instance);
+        filter.add(new PowerPredicate(ComparisonType.EQUAL_TO, 1));
+        filter.add(new ToughnessPredicate(ComparisonType.EQUAL_TO, 1));
+    }
+
+    public BessSoulNourisher(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{G}{W}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.addSubType(SubType.HUMAN, SubType.CITIZEN);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // Whenever one or more other creatures with base power and toughness 1/1 enter the battlefield under your control,
+        // put a +1/+1 counter on Bess, Soul Nourisher.
+        this.addAbility(new EntersBattlefieldControlledTriggeredAbility(
```

Comment:
this is a "one or more" trigger, so it should only trigger once when multiple creatures enter

Reply:
Then Welcoming Vampire (what I based this one) will also have to be fixed:
https://github.com/magefree/mage/blob/03cb3d45e3b8e4cbc636074753530694adee20f4/Mage.Sets/src/mage/cards/w/WelcomingVampire.java#L40-L41

Reply:
no, that one only triggers once per turn

Reply:
Fair enough, I guess the difference doesn't really matter